Mathlib/Data/Real/Sqrt.lean

```diff
@@ -450,6 +450,18 @@ theorem real_sqrt_le_nat_sqrt_succ {a : ℕ} : √(a : ℝ) ≤ Nat.sqrt a + 1 :
     exact le_of_lt (Nat.lt_succ_sqrt' a)
 #align real.real_sqrt_le_nat_sqrt_succ Real.real_sqrt_le_nat_sqrt_succ
 
+/-- The real square root is less than the natural square root plus one -/
+theorem real_sqrt_lt_nat_sqrt_succ {a : ℕ} : √(a : ℝ) < Nat.sqrt a + 1 := by
+  rw [sqrt_lt (by simp)] <;>
+  norm_cast
+  exact Nat.lt_succ_sqrt' a
+  exact Nat.le_add_left 0 (Nat.sqrt a + 1)
```

Comment:
When there are multiple goals like this, the best practice is to put `·` before each goal.